features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,330 @@
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A4' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160526                           |
+    En de persoon is vervolgens ingeschreven op een buitenlands adres met de volgende gegevens
+    | datum aanvang adres buitenland (13.20) | land adres buitenland (13.10) |
+    | 20181201                               | 6014                          |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20200415                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2010-07-01          |
+    | datumTot            | 2020-07-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | land.code | land.omschrijving            | straat      | adresseerbaarObjectIdentificatie | locatiebeschrijving         | datumAanvangAdreshouding | datumAanvangVolgendeAdreshouding | datumAanvangAdresBuitenland | datumAanvangVolgendeAdresBuitenland |
+    |           |                              | Luttestraat | 0800010000000002                 |                             | 20200415                 |                                  |                             |                                     |
+    | 6014      | Verenigde Staten van Amerika |             |                                  |                             |                          | 20200415                         | 20181201                    |                                     |
+    |           |                              |             |                                  | Woonboot bij de Grote Sloot | 20160526                 |                                  |                             | 20181201                            |
+    |           |                              | Laan        | 0800010000000001                 |                             | 20100818                 | 20160526                         |                             |                                     |
+
+Rule: Een verblijfplaats van een persoon met deels onbekende datum aanvang adreshouding/adres buitenland wordt geleverd als datumTot van de gevraagde periode groter of gelijk is aan de eerste dag van de deels onbekende datum aanvang adreshouding/adres buitenland en in het geval van een volgende adreshouding/adres buitenland is de datumVan kleiner dan datum aanvang volgende adreshouding/adres buitenland
```

Comment:
ook hier denk ik niet "groter of gelijk", omdat het een tot-datum is en geen tot-en-met-datum

Reply:
ik denk dat dit niet alleen geldt bij een deels onbekende datum aanvang, ook bij volledig onbekend. Dan is de eerste dag iets heel erg lang geleden, zoals 00000101

Reply:
rule aangepast aan 1e opmerking